web/src/components/IdeaList.vue

```diff
@@ -252,81 +256,109 @@ export default {
 </script>
 
 <style>
-#progress-bar {
+.progress-bar {
   background-color: var(--progress-bar-background);
   width: 100%;
-  height: 10px;
+  height: 1rem;
 }
-#progress-bar-progress {
+
+.progress-bar-progress {
   background-color: var(--primaryDark);
   height: 100%;
 }
-#ideas {
+
+.ideas {
   list-style-type: none;
-  margin: 0px;
-  padding: 0px;
+  margin: 0rem;
+  padding: 0rem;
 }
-#ideas li {
-  border-left: 8px solid transparent;
-  padding: 8px 16px 8px 16px;
+
+.ideas li {
+  border-left: 0.8rem solid transparent;
+  padding: 0.8rem 1.6rem 0.8rem 1.6rem;
 }
-#ideas li:hover {
+
+.ideas li:hover {
   background-color: var(--highlight);
   cursor: pointer;
 }
+
 .idea {
   display: flex;
   align-items: center;
   justify-content: space-between;
 }
-#idea-title {
+
+.idea__title {
   font-size: var(--primaryTextSize);
-  margin: 0px 0px 4px 0px;
+  margin: 0rem 0rem 0.4rem 0rem;
   text-overflow: ellipsis;
   overflow: hidden;
   white-space: nowrap;
-  padding: 0px;
+  padding: 0rem;
 }
-#idea-title > span {
-  font-size: 11px;
-  margin-left: 8px;
+
+.idea__title > span {
+  font-size: 1.1rem;
+  margin-left: 0.8rem;
   color: rgba(0, 0, 0, 0.54);
   background: white;
-  padding: 4px 8px;
-  border-radius: 4px;
+  padding: 0.4rem 0.8rem;
+  border-radius: 0.4rem;
   text-transform: uppercase;
 }
-.progress-undecided {
-  border-left: 8px solid var(--undecided) !important;
-}
-.progress-in-progress {
-  border-left: 8px solid var(--in-progress) !important;
-}
-.progress-done {
-  border-left: 8px solid var(--done) !important;
-}
-.badge {
+
+.idea__difficulty {
+  display: inline-block;
+  border-radius: 0.4rem;
   background-color: var(--primary);
   padding: var(--badgePadding);
   color: rgba(0, 0, 0, 0.54);
   font-size: var(--badgeTextSize);
+  text-transform: uppercase;
+}
+
+.progress--undecided {
+  border-left: 0.8rem solid var(--undecided) !important;
 }
+
+.progress--in-progress {
+  border-left: 0.8rem solid var(--in-progress) !important;
+}
+
+.progress--done {
+  border-left: 0.8rem solid var(--done) !important;
+}
+
 .idea-buttons {
   display: flex;
   justify-content: center;
   align-items: center;
 }
+
 .idea-menu-actions {
   border: none;
-  display: flex;
-  flex-flow: column;
+  display: flex !important;
+  flex-direction: column;
   padding: 0;
```

Comment:
This padding is overridden by popper. It overrides all of these styles because it seems to get included later and has the same specificity.